CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+### [0.17.4](https://github.com/CodeItQuick/open-sauced/compare/v0.17.1...v0.17.4) (2020-10-09)
+
+
+### Features
+
+* **add changelog.md capability:** npm run release now enabled ([97500fb](https://github.com/CodeItQuick/open-sauced/commit/97500fbe638fb2a617267b79d898d34e5a16b438)), closes [#668](https://github.com/CodeItQuick/open-sauced/issues/668)
+* **added husky:** husky for easy committing ([5d750b8](https://github.com/CodeItQuick/open-sauced/commit/5d750b8e4f559e9f41935817065848bcbfd24dca))
+* **adding commitizen versioning:** this adds commitzen and dependable commits ([011e864](https://github.com/CodeItQuick/open-sauced/commit/011e8647b521956071273b51d620e54a023e0bc4)), closes [#668](https://github.com/CodeItQuick/open-sauced/issues/668)
```

Comment:
Adding husky is a hard stop. It adds unnecessary friction to a project that gets contribution from all skill levels. The changelog commits will not need to be on the commit level since all merges happen as squashed commits. 

Same for commitzen it is not needed at the time of commit.

See this video from the semantic versioning maintainer:

https://youtu.be/HLhN2MxgWEM?list=PL0lo9MOBetEFmtstItnKlhJJVmMghxc0P&t=2515

The preference here should be updating the PR name to use the convention.

Reply:
Yeah, I'd like the way it was done in the video as well. I agree the commitizen+husky implementation could add unnecessary barriers to people contributing.

I'll integrate: https://github.com/zeke/semantic-pull-requests as semantic pull requests motto is to just include as much as possible, its also what is implemented in that video you gave. If you disagree let me know (at your convenience).

Reply:
link for my own reference: https://github.com/octokit/webhooks

Reply:
I've integrated https://github.com/amannn/action-semantic-pull-request which is a (very) simplified implementation of the probot library that just lints the PR title. @bdougie there is a box you need to check in settings outlined here: https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows/ that will allow pull_request_target to run. I attempted running without that as its likely preferred but I got an env failure.

If you would prefer to use the probot solution (zeke/semantic-pull-request), feel free to install and let me know that was the implementation you chose. Its a button the github maintainer has to click to allow access.

Reply:
Those settings are only for private repos. The use of `pull_request_target` has us covered. https://github.com/open-sauced/open-sauced/pull/850/files#diff-70c3a017bfdb629fd50281fe5f7ad22e29c0ddac36e7065e9dc6d4f0924104f4R3